.github/workflows/integration-tests.yml

```diff
@@ -28,6 +28,8 @@ on:
 
 jobs:
   integration_tests:
+    name: "Integration tests (${{ inputs.kube-version }}, ${{ inputs.java-version }}, ${{ inputs.it-category }}, 
```

Comment:
That is duplicated information which doesn't appear in the overview of the build outputs so we might as well remove it, imo